.github/workflows/checks.yml

```diff
@@ -35,38 +35,9 @@ jobs:
       run: mypy --python-version ${{ matrix.python }} -p build
 
 
-  flake8:
+  pre-commit: # per https://github.com/pre-commit/action
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Set up Python
-      uses: actions/setup-python@v1
-
-    - name: Install flake8
-      run: |
-        pip install --upgrade pip
-        pip install flake8
-        flake8 --version
-
-    - name: Run flake8
-      run: flake8 --show-source --statistics build
-
-  isort:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Set up Python
-      uses: actions/setup-python@v1
-
-    - name: Install isort
-      run: |
-        pip install --upgrade pip
-        pip install isort
-        isort --version
-
-    - name: Run isort
-      run: isort --check-only --diff --recursive build
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+    - uses: pre-commit/action@v2.0.0
```

Comment:
The current style declares human-readable names for each step, and has a new line between steps.

Reply:
I just used the suggestion of pre-commit, not sure how much value the human-readable names add, but if you really want can add them 👍